docs/TypeConstructor.md

```diff
@@ -6,75 +6,98 @@ value.
 
 @signature `Type`
 
-A constructor function can be provided that is called to convert incoming values set on this property, like:
+  A constructor function can be provided that is called to convert incoming values set on this property, like:
 
-```js
-{
-	prop: {
-		Type: Person
-	}
-}
-```    
+  ```js
+  import {DefineMap} from "can";
+  import {Person} from "//unpkg.com/can-demo-models@5";
 
-The `Type` constructor is called before the [can-define.types.type] property and before [can-define.types.set]. It checks if the incoming value
-is an [instanceof](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof) `Type`. If it is, or if it is `null` or `undefined`, it passes the original value through.  If not, it passes the original value to `new Type(originalValue)` and returns the
-new instance to be set.
+  const Example = DefineMap.extend({
+    prop: {
+      Type: Person
+    }
+  });
+  const ex = new Example();
+  ex.set( "prop", {first: "Justin", last: "Meyer"} );
```

Comment:
this should use `ex.prop = { ... }`.  Make sure we aren't using set unless we absolutely have to.